Automattic/woocommerce-payments
# Set newly created subscriptions to draft if onboarding is not complete.

Fixes #3378 
Fixes #3380 

#### Changes proposed in this Pull Request

Set subscription product status to draft when created before WCPay onboarding #3378

Subscriptions products published before before onboarding are completed are set to `draft`. Once the merchant completes onboarding all Subscriptions products that were set to draft will be published.

This change only captures subscriptions products that are being published prior to onboarding if the subscriptions extension is not activated. Specific product id's are saved as an option to be published automatically only if the status is still `draft` preventing incorrectly publishing other products (eg trash).

#### Testing instructions

On a fresh install, prior to onboarding, create a new subscription product and publish. The product should be in draft status.
Once onboarding to stripe is complete this subscription product should be published automatically.

-------------------


**Post merge**
